.github/renovate.json

```diff
@@ -1,42 +1,15 @@
 {
+  "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
     ":preserveSemverRanges",
     "config:base",
     "docker:enableMajor",
+    "helpers:pinGitHubActionDigests",
     "github>balena-io/renovate-config//.github/renovate"
```

Comment:
This is recursively including a copy of itself from master. This is why you are seeing things in the logs that were removed by this PR. I don't think this line should be here at all?

Reply:
It doesn't appear to make a difference, we still have `<prefix>` line in the logs.

Reply:
The line to remove is `"github>balena-io/renovate-config//.github/renovate"`.

You didn't add it, but it shouldn't be there.